src/main/java/com/chariotsolutions/meetupnotifier/meetup/api/MeetupGoogleEventTranslator.java

```diff
@@ -0,0 +1,47 @@
+package com.chariotsolutions.meetupnotifier.meetup.api;
+
+import com.google.api.client.util.DateTime;
+import com.google.api.services.calendar.model.Event;
+import com.google.api.services.calendar.model.EventDateTime;
+
+import com.chariotsolutions.meetupnotifier.meetup.api.messages.Result;
+
+
+import java.util.Date;
+
+public class MeetupGoogleEventTranslator {
```

Comment:
I would extract this into a "functional interface" - see https://docs.oracle.com/javase/8/docs/api/java/util/function/package-summary.html and http://radar.oreilly.com/2014/08/java-8-functional-interfaces.html.  

For example:
```java
public interface IResultConverter<T> {
    T convertTo(Result sourceResult)
}
```

Then you can use these in lambda expressions, etc.

Then you can create a `GoogleEventMeetupResultConverter` (or whatever you want to call it) that implements this interface.  This can be passed in as a method reference where a lambda expression of type `IResultConverter<Event>` is accepted. We can discuss the benefits of this further.